cli/src/ios/update.ts

```diff
@@ -96,7 +96,7 @@ export async function updatePodfile(
   const podfileLockPath = join(projectRoot, 'Podfile.lock');
   let podfileContent = await readFileAsync(podfilePath, 'utf8');
   podfileContent = podfileContent.replace(
-    /(Automatic Capacitor Pod dependencies, do not delete)[\s\S]*(# Do not delete)/,
+    /(Automatic Capacitor Pod dependencies, do not delete)[\s\S]*(# Do not delete)/,
```

Comment:
@jcesarmobile I noticed the [no-irregular-whitespace](https://eslint.org/docs/rules/no-irregular-whitespace) rule triggered for this space. Was it intentional to not use a normal space here?

I believe this change will have downstream effects when the podfile is regenerated in users' apps. (They'll see a git difference, which is probably fine.)

Reply:
I don't think it was intentional nor know how to create an "irregular" whitespace, but does the replace still work if you change it?

Reply:
Since it's next to the # I guess I pressed the space bar while still holding alt key when I added that line in the Podfile, and then I copied from the Podfile to the update.ts